epic_connection.py

```
@@ -0,0 +1,243 @@
+                success = True
+                break
+
+            except BrokenPipeError:
+                attempt += 1
+                logger.error(
+                    (
+                        "Failed to send message (probably due to connection "
+                        "reset on Epic side. Attempting to reconnect in 5 "
+                        f"minutes: {attempt}/5)"
+                    )
+                )
```

Ruff (0.14.7)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)